webpack/scenes/ContentViews/ContentViewSelectors.js

```diff
@@ -5,8 +5,7 @@ import {
 } from 'foremanReact/redux/API/APISelectors';
 import CONTENT_VIEWS_KEY from './ContentViewsConstants';
```

Comment:
Sorry I didn't catch this in the previous PR.. The value of the key should be uppercase snake case (CONTENT_VIEWS) to be consistent with the rest of our Redux actions:
![redux](https://user-images.githubusercontent.com/22042343/82101835-9df05900-96db-11ea-84d0-d11391b52efa.png)